src/vehicle_gui.cpp

```diff
@@ -1420,12 +1457,40 @@ void BaseVehicleListWindow::DrawVehicleListItems(VehicleID selected_vehicle, int
 		SetDParam(0, v->unitnumber);
 		DrawString(left, right, y + 2, str);
 
-		DrawVehicleProfitButton(v, vehicle_button_x, y + FONT_HEIGHT_NORMAL + 3);
+
```

Comment:
Empty line.

Reply:
This is in relation to the first commit, it is correct after the second commit.

Reply:
Fixed.